plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/OpenLineageListener.java

```diff
@@ -73,7 +73,8 @@ public class OpenLineageListener
     @Inject
     public OpenLineageListener(OpenLineageClient client, OpenLineageListenerConfig listenerConfig)
     {
-        this.client = requireNonNull(client);
+        this.client = requireNonNull(client, "client is null");
+        requireNonNull(listenerConfig, "listenerConfig is null");
```

Comment:
We don't use `requireNonNull` for config classes: https://github.com/trinodb/trino/pull/13940